src/ui/widgets/day.rs

```diff
@@ -0,0 +1,29 @@
+use super::*;
+
+pub struct DayWidget {
+    pub num: usize,
+    pub style: Style,
+}
+
+impl Default for DayWidget {
+    fn default() -> Self {
+        DayWidget {
+            num: 1,
+            style: Style::default().bg(Color::White).fg(Color::Black),
```

Comment:
for what is this "style"?
I change these values and nothing changes...

Reply:
Unnecessary ??

Reply:
Is it necessary. If you changed the values here it won't affect the appearance of the UI because in _src/ui/widgets/calendar.rs:58_ I explicitly set the style for each DayWidget